owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -158,8 +183,8 @@ open class FolderPickerActivity : FileActivity(),
         }
         // If current file is root folder
         else if (currentDirDisplayed.parentId == OCFile.ROOT_PARENT_ID) {
-            // If we are not in COPY mode, close the activity
-            if (pickerMode != PickerMode.COPY) {
+            // If we are not in COPY mode, or if we are in COPY mode and spaces are not allowed, close the activity
+            if (pickerMode != PickerMode.COPY || (pickerMode == PickerMode.COPY && !AccountUtils.isSpacesFeatureAllowedForAccount(baseContext, account, capabilities))) {
```

Comment:
Same here, we can rely on the `spaceId` existence. What do you think?

```suggestion
            if (pickerMode != PickerMode.COPY || (pickerMode == PickerMode.COPY && currentDirDisplayed.spaceId == null)) {
```

Reply:
💯